akka-docs-dev/rst/scala/code/docs/stream/StreamPartialFlowGraphDocSpec.scala

```diff
@@ -142,4 +144,62 @@ class StreamPartialFlowGraphDocSpec extends AkkaSpec {
 
     Await.result(matSink, 300.millis) should equal(1 -> "1")
   }
+
+  "help users in providing complex ports" in {
+    object AudioPorts extends Ports {
+      val nums = Input[Int] // TODO: use I / O words or keep sinks/sources?
```

Comment:
(probably should be same as in fleximerge if we want this style of thing - so `createInputPort`?)